assets/js/order-script.js

```diff
@@ -27,37 +30,37 @@ jQuery(document).ready(function($) {
 
 			window.open(url,'_blank');
 		}
-	});
+	} );
 
-	$('#wpo_wcpdf-data-input-box').insertAfter('#woocommerce-order-data');
+	$( '#wpo_wcpdf-data-input-box' ).insertAfter('#woocommerce-order-data');
 	
 	// enable invoice number edit if user initiated
-	$( ".wpo-wcpdf-set-date-number, .wpo-wcpdf-edit-date-number, .wpo-wcpdf-edit-document-notes" ).on("click", function() {
-		$form = $(this).closest('.wcpdf-data-fields-section');
-		if ($form.length == 0) { // no section, take overall wrapper
-			$form = $(this).closest('.wcpdf-data-fields');
-		}
+	$( '#wpo_wcpdf-data-input-box' ).on( 'click', '.wpo-wcpdf-set-date-number, .wpo-wcpdf-edit-date-number, .wpo-wcpdf-edit-document-notes', function() {
+		$form = $(this).closest('.wcpdf-data-fields');
```

Comment:
you've applied variable scoping using `let` in several changes above, is there a reason you haven't scoped this variable explicitly? Undeclared, it's a global variable by default, but you need to take into account that the browser may run in strict mode.